resources/dashboards/Capture-Dashboard.json

```diff
@@ -0,0 +1,11 @@
+{
+   "title": "Capture Dashboard",
```

Comment:
This was messed up in the end, but it is now fixed. It used to pull the Analyze Table, and not it pulls the Capture Table